webnn/META.yml

```diff
@@ -0,0 +1 @@
+spec: https://webmachinelearning.github.io/webnn/
```

Comment:
Could you add some suggested reviewers to this file?

Reply:
Thanks @Honry.

@anaskim @dontcallmedom @Honry May I invite you to be reviewers? Then I would add your name into suggested reviewers list of META.yml file, thanks.

Reply:
Sure. :)